pkg/controllers/rollout/suite_test.go

```diff
@@ -91,6 +94,20 @@ var _ = BeforeSuite(func() {
 	})
 	Expect(err).Should(Succeed())
 
+	RestMapper := mgr.GetRESTMapper()
+	Expect(RestMapper).NotTo(BeNil())
+	validator.RestMapper = RestMapper
+	validator.ResourceInformer = &informer.FakeManager{
+		APIResources: map[schema.GroupVersionKind]bool{
+			{
+				Group:   "",
+				Version: "v1",
+				Kind:    "Namespace",
+			}: true,
+		},
+		IsClusterScopedResource: true,
+	}
```

Comment:
just curious why we need these. if i member it correctly, rollout controller does not call webhook?

Reply:
The rollout controller integration test [creates a crp](https://github.com/Azure/fleet/blob/3c13e7d99cf58f6a7e34a55ec559424fceb993b0/pkg/controllers/rollout/controller_integration_test.go#L66)

Reply:
nope, we even did not create the webhook sever.

We need something like  https://github.com/Azure/fleet/blob/main/cmd/hubagent/main.go#L111 when creating the manager.

Reply:
As discussed offline, webhook is being called in rollouts controller.go file which is why the integration test was failing previously. Need to revisit and discuss if needed here.

Reply:
statement is not accurate, > "validation is being called", not the webhook

When you use the "webhook" terminology, it's talking about 
[k8 webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#webhook-request-and-response)